layer/resource_manager.py

```diff
@@ -1,228 +0,0 @@
-                total_num_files += 1
-                total_files_size_bytes += os.path.getsize(os.path.abspath(local_path))
-                upload_task = self._upload_resource(
-                    project_full_name=project_full_name,
-                    function_name=function.func_name,
-                    resource_path=ResourcePath(path=local_path),
-                    file_path=os.path.abspath(local_path),
-                    session=session,
-                    state=state,
-                )
-                upload_tasks.append(upload_task)
-        state.total_num_files = total_num_files
-        state.total_resource_size_bytes = total_files_size_bytes
-
-        asset_type = function.asset_type
-        asset_name = function.asset_name
-        if asset_type == AssetType.DATASET:
-            tracker.mark_dataset_resources_uploading(asset_name, state)
-        elif asset_type == AssetType.MODEL:
-            tracker.mark_model_resources_uploading(asset_name, state)
-        await asyncio.gather(*upload_tasks)
-        if asset_type == AssetType.DATASET:
-            tracker.mark_dataset_resources_uploaded(asset_name)
-        elif asset_type == AssetType.MODEL:
-            tracker.mark_model_resources_uploaded(asset_name)
-
-    def wait_resource_upload(
-        self,
-        project_full_name: ProjectFullName,
-        functions: List[FunctionDefinition],
-        tracker: RunProgressTracker,
-    ) -> None:
-        """
-        Collect and upload local files as resources for all functions decorated with `@resources`.
-        """
-        self._log.debug("Uploading resources")
-        # run in a separate thread to make sure no other loop is running
-        with ThreadPoolExecutor(max_workers=1) as executor:
-            upload = executor.submit(
-                lambda: asyncio.run(
-                    self._upload_resources(project_full_name, functions, tracker)
-                )
-            )
-            upload.result()
-
-    async def _download_resource(
-        self, download_url: str, target_dir: str, session: aiohttp.ClientSession
-    ) -> Any:
-        download_url_path = urlparse(download_url).path
-        relative_resource_part = _strip_resource_root_path(download_url_path)
-        resource_dir, resource_file = os.path.split(relative_resource_part)
-        download_dir = os.path.join(target_dir, resource_dir)
-        Path(download_dir).mkdir(parents=True, exist_ok=True)
-        download_path = os.path.join(download_dir, resource_file)
-        async with session.get(
-            download_url, timeout=aiohttp.ClientTimeout(total=None)
-        ) as response:
-            with open(download_path, mode="wb") as f:
-                bytes_total = 0
-                async for data in response.content.iter_chunked(8192):
-                    bytes_total += f.write(data)
-        self._log.info(
-            f"Downloaded {download_url_path} to {download_path}, bytes total: {bytes_total}"
-        )
-
```

Comment:
Where did this logic go?!

Reply:
It is no longer used since resources are embedded into the executable zip now